react-components/src/metadata/MetadataTemplateView.js

```diff
@@ -236,7 +259,10 @@ class MetadataTemplateAttributeView extends Component {
                                 const hasAVUs = avuFields && avuFields.filter(avuField => avuField).length > 0;
 
                                 return ((writable || hasAVUs) &&
-                                    <ExpansionPanel key={attribute.name} defaultExpanded={hasAVUs}>
+                                    <ExpansionPanel key={attribute.name}
+                                                    expanded={!!expanded[attribute.name]}
```

Comment:
The `!!expanded[attribute.name]` here forces `expanded` to a boolean value, even if a value as not yet been set for `attribute.name` in the `expanded` state map. Otherwise, if a `null` is passed as the `expanded` prop, then `ExpansionPanel` will assume it is not controlled and will manage its own internal `expanded` state.